### ISSUE TITLE
[Feature Request] Admin party chat spy

### ISSUE THREAD

**Issue by SXRWahrheit** [mcMMO-Dev/mcMMO#3340]
As the name implies. I'd like to be able to see what's being said in mcmmo party chat if I have a specified permissions node.

**Comment by Flavourized** [comment]
This feature has been requested in 7 separate tickets and countless amount of users. 

Devs don't give a shit about it and they won't ever implement it (even though it's been requested a stupid amount of times!). 

**See the following:** 

[Feb 07, 2013] @TfT-02 said "Party chat spy is actually not a bad suggestion, we might consider adding that because that can be very helpful." in issue [#647](https://github.com/mcMMO-Dev/mcMMO/issues/647).

[Apr 28, 2013] @TfT-02 said "We've decided that we will not be implementing this feature. We feel that adding this functionality does not have any benefits and that it is too intrusive. We hope you understand this decision." in issue [#956](https://github.com/mcMMO-Dev/mcMMO/issues/956).

[Oct 28, 2013] @Glitchfinder said "Closing, but leaving marked as on hold for future reference. We may come back to this one." in issue [#647](https://github.com/mcMMO-Dev/mcMMO/issues/647).

[Feb 12, 2014] @TfT-02 said "Has been requested a long time ago. See #124, it has now been put on hold since it's not really on of our priorities at the moment." in issue [#1845](https://github.com/mcMMO-Dev/mcMMO/issues/1845).

[Jul 22, 2014] @TfT-02 said "Has been requested a long time ago. See #124, it has now been put on hold since it's not really on of our priorities at the moment." in issue[ #2145](https://github.com/mcMMO-Dev/mcMMO/issues/2145).

[Mar 12, 2016] @Sikatsu requested "anyway to enable socialspy for the party chat? Or could you create one?" in ticket [#2811](https://github.com/mcMMO-Dev/mcMMO/issues/2811) which went unanswered and is still open.

[Apr 06, 2016] @SioWolf provided **very** valid arguments (in issue #[2888](https://github.com/mcMMO-Dev/mcMMO/issues/2888)) to @TfT-02's insight in issue [#2145](https://github.com/mcMMO-Dev/mcMMO/issues/2145).

---------------------------

So even though we have multiple server owners and devs - who have asked for this feature to be implemented every year for the past 4 years - asking for this feature, it never seems to make it to the "priority list" of the devs.

My question would be "When does a feature request become a 'priority'?"

**Comment by Glitchfinder** [comment]
So this is the first time I've poked my head into mcMMO-related things for a while. (Literally, in years), and the only reason I'm doing so here is because I was specifically pinged and thus it showed in my github notifications when I logged in. I'll start by answering your last question, the bit about when something becomes a priority.

The short answer to that is, it doesn't. The long answer is that, based on the recent commit history, updates and changes to mcMMO are running on fumes right now, mostly being powered by @t00thpick1. This plugin has always been worked on in the spare time of the devs, and as best I can tell, it looks like there is very little of that to go around. I personally dropped out of active development a couple of years ago, and checking my email right now, I have somewhere in the range of seven thousand emails from this issue tracker alone. I simply don't have time to work on projects like this anymore, and as best I can tell, that seems to not just be me.

As for what makes a feature, even one acknowledged as a good idea, get implemented? Honestly, that's at the whims of whoever is writing code at the time. When I was actively working on the project, my major priorities were cleanup, organization, and updating. You'll see a big chunk of the bugs on the tracker around 2012ish were directly addressed by myself, as well as a great deal of other backend issues. On the other hand, there are some things that can't be done without a cooperative effort. For example, the oldest open issue on the entire tracker is one I made a concerted effort to personally address, only to have major updates to the plugin completely undo my work towards making it reasonable.

Can this feature get added to the plugin? Yes, it can. Can I do so? Yes, if I want to go to work tomorrow distinctly lacking sleep and with a nasty headache. At the moment, your best bet is t00thpick1, who seems to have more free time and inclination to work on the plugin.

In short, though, it's not a matter of whether or not we care about something. We all have to balance our work and personal lives, and if I want enough money to keep a roof over my head, that means not working on projects like this.
